src/Components/Admin/common/itemCard.js

```diff
@@ -17,6 +17,20 @@ const useStyles = makeStyles(() => ({
         maxWidth: 400,
         marginTop: 16,
     },
+    metadata: {
+        display: "flex",
+        justifyContent: "space-between",
+        marginLeft: "5.7%",
+        marginRight: "5.7%",
+        marginTop: -4,
+        backgroundColor: "white",
+        borderBottomLeftRadius: "5px",
+        borderBottomRightRadius: "5px",
```

Comment:
Nit: the designs in figma appear to have a larger border radius here, similar to what is on the top left and right of the image.